adapters/Mintegral/MintegralAdapter/Bidding/GADMAdapterMintegralNativeLoader.h

```diff
@@ -0,0 +1,23 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <Foundation/Foundation.h>
+#import <GoogleMobileAds/GoogleMobileAds.h>
+
+@interface GADMAdapterMintegralNativeLoader : NSObject
```

Comment:
Can you rename this class to `GADMAdapterMintegralNativeAdLoader`? To be consistent with the other classes.

Reply:
thanks, i have fixed it.